package com.example.crud.exception;

public class ConflictException extends RuntimeException{
    public ConflictException(String mensagem) {
        super(mensagem);
    }
}
